aws/ci_build

```diff
@@ -45,7 +45,7 @@ def upload_log_and_get_link(key, body, metadata)
         body,
         {metadata: metadata}
       )
-  " <a href='#{log_url}'>☁ Log on S3</a>"
+  " <a href='#{log_url}'>  Log on S3</a>"
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1615761/25585544/05d70002-2e50-11e7-904c-07ae8e6378f8.png)

It's mostly for fun. I do think there's a little value in adding iconography to our Slack interface if it's not too noisy.  I build subtle associations with these things (like tableflip guy when tests fail) so that, personally, I find things scan easier than if we just had a wall of text.